src/styles/scoresaber.scss

```diff
@@ -411,3 +411,18 @@ hr {
    border-bottom-color: var(--muted);
    color: var(--muted);
 }
+
+@media only screen and (max-width: 512px) {
+   .section {
+      padding-left: 1rem;
+      padding-right: 1rem;
+   }
+
+   .window {
+      padding: 1rem .5rem;
```

Comment:
Very small nitpick, I think the horizontal padding should increased slighlty to 0.7 or 0.8 rem just to avoid text being so close to the edges

Reply:
I reduced it so much because I didn't notice anywhere that the text was embedded directly into this container. It's always still a separate container with its own padding, like this:
![image](https://user-images.githubusercontent.com/1758707/143187211-5fdad527-b60f-46d8-a2c3-f9802669cfd0.png)

Reply:
The main place I noticed it, was on the leaderboards/maps page with the filter boxes as that has content directly in it without its own padding.

Before:
![image](https://user-images.githubusercontent.com/55684874/143187510-80043fe8-c285-4319-820e-65ebd2d67e8a.png)

Yours:
![image](https://user-images.githubusercontent.com/55684874/143187691-b4188773-638e-4031-a41e-94fa20d9d9a0.png)

imo it's just a little too close to the edge and increasing it slightly gives it that small buffer between the edge and the text, while still making it slimmer than before.
Alternatively, you could go and add padding to those filters specifically but I think just a small global change is nicer.

Reply:
Ok, I'll increase that a little bit. To .75rem? Possibly more targeted in some places, like on player scores.

Reply:
Yeah sounds good, I agree some places like player scores can benefit slightly from the reduced padding but that can probably be applied more locally as opposed to globally.

Reply:
@Dannypoke03  I merge main to this branch so I don't have to close this PR. Or should I close it, rebase the branch and make a new PR?

Reply:
Now it looks good imo 
![image](https://user-images.githubusercontent.com/1758707/143289967-7a014d92-ea2f-43ca-ba7e-1baf650f34a5.png)